sdk/servicebus/service-bus/test/utils/testutils2.ts

```diff
@@ -456,17 +455,18 @@ async function purgeForTestClientType(
 
   if (entityPaths.queue) {
     receiver = serviceBusClient.createReceiver(entityPaths.queue, "receiveAndDelete");
-    deadLetterReceiver = serviceBusClient.createDeadLetterReceiver(entityPaths.queue, {
-      receiveMode: "receiveAndDelete"
+    deadLetterReceiver = serviceBusClient.createReceiver(entityPaths.queue, {
+      receiveMode: "receiveAndDelete",
+      subQueue: "deadLetter"
```

Comment:
This looks pretty clean TBH. I'll miss our named dead letter receiver names for it being more "in your face" but I think this is a nice replacement.